### DISCUSSION TITLE
Does it store what I type with it?

### DISCUSSION THREAD

**Discussion by tusharkhan5607** [OpenBangla/OpenBangla-Keyboard#194] [Q&A] [answered]
Hey, I am a little bit concerned with keylogging issue!
Does this app record what I type? Do you guys keep any logs?

**Comment by ahmubashshir** [comment]
AFAIK, It will only store your custom word maps and your choice on selection.

```sh
ls -Rla ~/.local/share/openbangla-keyboard
```

**Comment by mominul** [comment, marked as answer]
No, OpenBangla Keyboard doesn't keep any logs of your typing. However, it only saves the word you have chosen yourself (the non-default one) from the suggestion list in Avro Phonetic mode. OBK keeps it as a reference purpose to help you when you type the same word in the future so you don't have to select the non-default candidate again. OBK will select the word by default which you had selected yourself previously. And the file is locally saved in your home directory. You can see it yourself in the directory path @ahmubashshir has given.

I think this is enough to resolve your concern, thanks!

**Comment by bdeshi** [comment]
@h-sifat also, the stored custom suggestions are never transferred to anywhere else from your computer.
